packages/wallets/src/components/TradingAccountCard/TradingAccountCard.tsx

```diff
@@ -1,19 +1,17 @@
-import React, { ReactElement } from 'react';
-import useDevice from '../../hooks/useDevice';
+import React from 'react';
 import './TradingAccountCard.scss';
 
 type TProps = {
     description: string;
-    icon: ReactElement;
+    icon: React.ReactNode;
```

Comment:
So we go with React.ReactNode instead of ReactElement? just wanna clarify the standard

Reply:
Better to go with reactNode cause its more flexible in terms of children and it doesnt get executed if the function is not fired